admin-ui/app/utils/ApiUtils.ts

```diff
@@ -39,6 +39,23 @@ export function handleTypedResponse<T>(
   }
 }
 
+export function handleResponseTyped<T>(
```

Comment:
handleResponseTyped duplicates timeout/error handling logic found in other response helpers. Consider consolidating these functions to reduce redundancy.